.eslintrc

```diff
@@ -1,38 +0,0 @@
-{
-  "parser": "@typescript-eslint/parser",
-  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:prettier/recommended"],
-  "parserOptions": {
-    "ecmaVersion": 2018,
-    "sourceType": "module"
-  },
-  "env": {
-    "browser": true
-  },
-  "plugins": ["unicorn", "header"],
-  "rules": {
-    "@typescript-eslint/ban-ts-comment": ["error", { "ts-expect-error": "allow-with-description" }],
-    "@typescript-eslint/interface-name-prefix": "off",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-non-null-assertion": "off",
-    "unicorn/filename-case": "error",
-    "curly": "error",
-    "eqeqeq": "error",
-    "no-return-await": "error",
-    "require-await": "error",
-    "header/header": [
-      "error",
-      "line",
-      [" Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.", " SPDX-License-Identifier: Apache-2.0"]
-    ]
-  },
-  "overrides": [
```

Comment:
is this no longer needed?

Reply:
Correct. Two changes happened

1. They changed root+overrides structure to a flat array of configs, each operates as an override for the given `files`. More details here: https://eslint.org/docs/latest/use/configure/migration-guide#glob-based-configs
2. We only needed an override for postcss.js file because of `process` global var. I replaced it with an import, which is cleaner anyway